docs/configuration/trap-configuration.md

```diff
@@ -143,4 +143,15 @@ Example:
 ```yaml
 traps:
   aggregateTrapsEvents: "true"
+```
+
+### Updating trap configuration
+If you need to update part of traps configuration, you can do it by editing the `values.yaml` and then running below command to restart the pod deployment.
+```
+microk8s kubectl rollout restart deployment snmp-splunk-connect-for-snmp-trap -n sc4snmp
+```
+
+NOTE: Name of deployment can differ based on helm name and can be checked with:
```

Comment:
`the deployment`, and I'd write `helm installation name` because it's somethign that user sets with the helm upgrade command

Reply:
done